src/schema/v2/me/collection.ts

```diff
@@ -100,7 +100,7 @@ export const CollectionType = new GraphQLObjectType<any, ResolverContext>({
         "Name of the collection. Has a predictable value for 'standard' collections such as Saved Artwork, My Collection, etc. Can be provided by user otherwise.",
       resolve: ({ name }) => {
         if (name === "Saved Artwork") {
```

Comment:
**suggestion**: Could this instead check the `saves` and `default` attributes to determine when to override the name? If `saves=true` and `default=true`, I think that means we should use the default collection name of "Saved Artworks", regardless of what the underlying database name is.

Reply:
Added changes (https://github.com/artsy/metaphysics/pull/4947/commits/b0ea60241efe6499ac4187cdaa5918ae43496a5e)

Reply:
You can check this by executing a similar GraphQL query for any user. There should always only be one collection returned, the "standard" collection:

```graphql
{
  me {
    internalID
    collectionsConnection(first: 5, saves: true, default: true) {
      edges {
        node {
          default
          saves
          name
        }
      }
    }
  }
}
```